tests/override.phpt

```diff
@@ -6,14 +6,9 @@ fileinfo
 --INI--
 shadow.override=imagepng@w1
 --SKIPIF--
-<?php if (!extension_loaded('shadow')) {
+<?php if (!extension_loaded('shadow') || !extension_loaded('gd')) {
```

Comment:
Please consider this comment from Jira:

>> test [override.phpt](https://github.com/sugarcrm/shadow/blob/master/tests/override.phpt) is skipped even with the appropriate libraries on the system, I believe in part because it uses the [`dl`](http://php.net/manual/en/function.dl.php) function

> From what I understand, there's no environment under which this test won't be skipped. AFAIK, `gd` can only be installed as a shared extension. At least, all Linux distros provide it as a separate package. At the same time, `make test` runs `php -n` and therefore it won't be able to dynamically load any other extension than the one which is being tested (because the `extension_dir` where `gd` is located is ignored).

> Even if `gd` is available, this test looks unfinished and won't pass. It should be either reworked to use some core functions or removed entirely.

Unless there is a way to make this test not skip, there's no point in trying to improve it. Maybe the previously existing `dl('gd.so')` was intended to dynamically load the extension.

Reply:
This test actually passes on Travis. It can be also passed locally if we follow the same setup instructions - install php version via `phpenv` and run `sed` command `sed -i.stock -e "s/\$ini_settings\['extension'\]\[\] = \$ext_dir \. DIRECTORY_SEPARATOR \. \$req_ext \. '\.' \. PHP_SHLIB_SUFFIX;/array_unshift\(\$ini_settings\['extension'\], \$ext_dir \. DIRECTORY_SEPARATOR \. \$req_ext \. '\.' \. PHP_SHLIB_SUFFIX\);/" run-tests.php`
That command patches test runner so shadow extension is loaded at the very end as expected. The same `sed` is required to run test with `zip` extension.
I completely agree it's better to rework these tests to depend on some standard non-shared extension. However, the first goal I suppose is to make it 7.3 compatible.